### PR TITLE
Added prototypes used for HTML scraping

### PR DESCRIPTION
**Pull request by lvass74**
Added prototypes created for HTML scraping (from the project of David Vízi)

### CODE REVIEW

**Review by csabee1979 — APPROVED** [review]

File: html/xpath value finder/internal/35629a5f-aad8-4c08-8946-7a46af449595.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      }
+    },
+    "outputs": {
+      "25eed9f7-7c3f-45f6-890e-0bbd32ab803a": {
+        "type": "any",
+        "name": "results"
+      }
+    }
+  },
+  "name": "html/XPath value finder/Internal",
```
Comment (danstocker) [inline html/xpath value finder/internal/35629a5f-aad8-4c08-8946-7a46af449595.json:39]: Shouldn't this be called "..../XPathe value finder (internal)"?

File: string/global regex matcher/internal/70ea8986-7c87-49bb-aa42-c2d1b970c7d4.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "es6-node": {
+      "cleanupFunction": "",
+      "dependencies": {},
+      "initFunction": "",
+      "inputFunctions": {
+        "19476252-601a-47a3-bb8e-78e5b6661233": "const result = new RegExp(data.regex, 'g').exec(data.string);\nconst [matched, ...groups] = result || [];\noutputs.matches(!!matched, tag);\noutputs.groups(groups, tag);"
+      }
+    }
+  },
+  "name": "string/Global regex matcher/Internal",
```
Comment (danstocker) [inline string/global regex matcher/internal/70ea8986-7c87-49bb-aa42-c2d1b970c7d4.json:35]: Same re. including "internal" in name

File: html/xpath value finder/internal/35629a5f-aad8-4c08-8946-7a46af449595.json (hunk 1 above)
Comment (lvass74) [reply html/xpath value finder/internal/35629a5f-aad8-4c08-8946-7a46af449595.json]: Fixed

File: string/global regex matcher/internal/70ea8986-7c87-49bb-aa42-c2d1b970c7d4.json (hunk 2 above)
Comment (lvass74) [reply string/global regex matcher/internal/70ea8986-7c87-49bb-aa42-c2d1b970c7d4.json]: Fixed

**Review by danstocker — APPROVED** [review]

**Pull request merged**